ajax/libs/jquery.loadTemplate/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "jquery.loadTemplate",
+    "description": "A template plugin for jQuery. Allows templating without cluttering JavaScript code with markup",
+    "filename": "jquery.loadTemplate.min.js",
+    "keywords": [
+       "templates",
+       "templating",
+       "jquery-plugin"
+    ],
+    "author": "codepb",
+    "homepage": "http://codepb.github.io/jquery-template/",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com:codepb/jquery-template.git"
+    },
+    "npmName": "jquery.loadtemplate ",
+    "npmFileMap": [
```

Comment:
Let's use git auto-update.
Because there are more versions release from GitHub(54) than npm(3).
The more versions can let web front-end developer more choose.
Thank you.